src/components/atoms/OcDrop/OcDrop.vue

```diff
@@ -207,6 +207,26 @@ export default {
   .tippy-content {
```

Comment:
Would it make sense to move it to `OcList` and add a property to enable hover?

Reply:
Then it'd hover also on lists that are not inside an OcDrop, not sure if that behavior is wanted ;) since this PR (together with the one in `web`) is about aligning the design of links&buttons (and their hover styles) in dropdowns it may be too much

Reply:
I created an extra class "action-menu-item-hover" within ocDrop that can be applied on wish to `<li>` or `<action-menu-item>` within ocDrop. Applied in web pr to menus of "New", "Upload" and context menu https://github.com/owncloud/web/pull/6762/commits/cc094981c4d2f1e4b3de602e3c5abe22c9e741fa